src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ActionRow, Button, Stack, Form, Hyperlink,
+} from '@edx/paragon';
+
+import { useActiveQueryParams } from './data';
+
+function UserEnrollmentForm({ className }) {
+  const [isTermsChecked, setIsTermsChecked] = useState(false);
+  const activeQueryParams = useActiveQueryParams();
+
+  return (
+    <form className={className}>
+      <div className="mb-4">
+        <h3 className="h4 mb-3">Personal information</h3>
+        <Stack direction="horizontal" gap={3}>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
```

Comment:
what happens if we don't specify a no-op for all these `onChanges`?

Reply:
Nothing 😆 . It's just an artifact or starting to go down the path of making the form function and then realizing that should be a separate PR/task. Because we will likely need the `onChange` when we make the form functional, I left them in as no-ops for now.